server/src/capture.rs

```diff
@@ -0,0 +1,227 @@
+// src/capture.rs
+
+// ## Imports
+//
+// ### Standard library
+use std::fs;
+use std::io;
+use std::path::Path;
+use std::sync::Arc;
+
+// ### Third-party
+use chrono::Local;
+use log::{error, info};
+use serde::{Deserialize, Serialize};
+use tokio::sync::Mutex;
+use tokio_postgres::{Client, NoTls};
+
+// ### Local
+
+/*
+The `Event` struct represents an event to be stored in the database.
+
+Fields:
+    - `user_id`: The ID of the user associated with the event.
+    - `event_type`: The type of event (e.g., "keystroke", "file_open").
```

Comment:
Do you want to make `event_type` an enum? String = flexibility, but constraints can be helpful.